src/types/index.ts

```diff
@@ -324,9 +353,9 @@ export interface IMiddlewareCustomMethods {
   fetchByPath: (p: string) => Promise<any>
 }
 
-export interface IMiddleware extends
-  IMiddlewareClient,
-  IMiddlewareCustomMethods {}
+export interface IMiddleware extends IMiddlewareClient,
+  IMiddlewareCustomMethods {
+}
```

Comment:
I don't understand this change. This was a list of extended interfaces split into lines. Current change makes the code less readable.